Tools/CISupport/ews-build/steps_unittest.py

```diff
@@ -5044,7 +5044,8 @@ def test_failure(self):
   File "/usr/lib/python2.7/os.py", line 382, in _execvpe
     func(fullname, *argrest)
 OSError: [Errno 2] No such file or directory'''),
-            ExpectShell(command=['system_profiler', 'SPSoftwareDataType', 'SPHardwareDataType'], workdir='wkdir', timeout=60, logEnviron=False) + 0,
+            ExpectShell(command=['system_profiler', 'SPHardwareDataType'], workdir='wkdir', timeout=60, logEnviron=False) + 0,
```

Comment:
Seems like removal of `SPSoftwareDataType` here was unintentional (and causing ews unit-test failure).